cli.js

```diff
@@ -71,6 +71,11 @@ const openGitHub = async name => {
 			process.exit(1);
 		}
 
+		if (error instanceof PackageNotFoundError) {
+			console.error(`Package \`${name}\` doesn't exist!`);
+			process.exit(1); // TODO: this is 'fail-fast' -> if invoked with multiple names, later ones won't open
+		}
```

Comment:
Should this exit here, or just log the message and skip this package?

Reply:
Skip this package, log, and continue.